test/moduleView.js

```diff
@@ -0,0 +1,25 @@
+/* global describe,it */
+'use strict';
+
+var assert = require('assert'),
+  nemoFactory = require('nemo-mocha-factory'),
+  plugins = require('./plugins'),
+  nemo = {},
+  setup = {
+    'view': [
+      {
+        'name': 'login',
+        'locator': 'module:nemo-shared-objects/login'
+      }
+    ]
+  };
+
+describe('nemo-view @moduleViewSuite@', function () {
+  nemoFactory({'context': nemo, 'plugins': plugins, 'setup': setup});
+  it('should find the PayPal login email locator', function (done) {
+    var emailLocator = nemo.view.login.emailBy();
+    assert.deepEqual(emailLocator, {'using': 'id', 'value': 'login_email'});
+    done();
+  });
```

Comment:
some of the test with PayPal homepage are failing because it looks like now by default you don't see email/password text boxes when navigating to paypal.com via browser like below. I think switching to sample heroku app would be better for unit tests.

![image](https://cloud.githubusercontent.com/assets/3066339/6516233/214bd9fc-c345-11e4-8a32-e9ca899c9f3e.png)